src/styles/utilities/variables.scss

```diff
@@ -0,0 +1,8 @@
+$card-width: 200px;
+$secondary-font-size: 10px;
+$secondary-line-height: 14px;
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$Blue-accent: #00acdc;
+$transition-duration: 300ms;
```

Comment:
[LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards. I didn't see this function